webapp/src/components/expanded_view/component.tsx

```diff
@@ -47,7 +46,7 @@ import {
 
 import './component.scss';
 
-const EMOJI_VERSION = 13;
+const EMOJI_VERSION = '13';
```

Comment:
:facepalm:

Reply:
Ah, but the actual problem was the `data` prop, right?

Reply:
Ah yes. I just made this explicitly a string since we can pass versions such as 13.1 to it as well.